glib/src/gobject/flags.rs

```diff
@@ -3,24 +3,30 @@
 use crate::translate::*;
 
 bitflags::bitflags! {
+    #[doc(alias = "GParamFlags")]
     pub struct ParamFlags: u32 {
-        const READABLE = 1;
-        const WRITABLE = 2;
-        const READWRITE = 3;
-        const CONSTRUCT = 4;
-        const CONSTRUCT_ONLY = 8;
-        const LAX_VALIDATION = 16;
-        const USER_0 = 128;
-        const USER_1 = 256;
-        const USER_2 = 1024;
-        const USER_3 = 2048;
-        const USER_4 = 4096;
-        const USER_5 = 8192;
-        const USER_6 = 16384;
-        const USER_7 = 32768;
-        const USER_8 = 65536;
-        const EXPLICIT_NOTIFY = 1073741824;
-        const DEPRECATED = 2147483648;
+        #[doc(alias = "G_PARAM_READABLE")]
+        const READABLE = gobject_ffi::G_PARAM_READABLE as u32;
+        #[doc(alias = "G_PARAM_WRITABLE")]
+        const WRITABLE = gobject_ffi::G_PARAM_WRITABLE as u32;
+        #[doc(alias = "G_PARAM_READWRITE")]
+        const READWRITE = gobject_ffi::G_PARAM_READWRITE as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT")]
+        const CONSTRUCT = gobject_ffi::G_PARAM_CONSTRUCT as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT_ONLY")]
+        const CONSTRUCT_ONLY = gobject_ffi::G_PARAM_CONSTRUCT_ONLY as u32;
+        #[doc(alias = "G_PARAM_LAX_VALIDATION")]
+        const LAX_VALIDATION = gobject_ffi::G_PARAM_LAX_VALIDATION as u32;
+        #[doc(alias = "G_PARAM_STATIC_NAME")]
+        const STATIC_NAME = gobject_ffi::G_PARAM_STATIC_NAME as u32;
```

Comment:
This makes no sense from Rust, same for the other `G_PARAM_STATIC_...` flags

Reply:
Ok§